pkg/freeze/pause.go

```diff
@@ -0,0 +1,80 @@
+package freeze
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"knative.dev/container-freezer/pkg/freeze/common"
+	"knative.dev/container-freezer/pkg/freeze/containerd"
+	"knative.dev/container-freezer/pkg/freeze/crio"
+)
+
+const (
+	runtimeTypeContainerd = "containerd"
+	runtimeTypeCrio       = "crio"
+)
+
+type CRI interface {
+	List(ctx context.Context, podUID string) ([]string, error)
+	Pause(ctx context.Context, container string) error
+	Resume(ctx context.Context, container string) error
+}
+
+type CRIImpl struct {
```

Comment:
nit: is there another name we could use here? Maybe `ContainerRuntime`? (I'm not good at naming btw...)

Reply:
rename to ContainerRuntimeImpl